frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { getIcon } from "__support__/ui";
+
+import {
+  createMockCard,
+  createMockDashboardOrderedCard,
+  createMockActionDashboardCard,
+  createMockMetadata,
+} from "metabase-types/api/mocks";
+
+import { DashCardCardParameterMapper } from "./DashCardCardParameterMapper";
+
+const setup = options => {
+  const metadata = createMockMetadata();
+  render(
+    <DashCardCardParameterMapper
+      card={createMockCard()}
+      dashcard={createMockDashboardOrderedCard()}
+      editingParameter={{}}
+      target={null}
+      mappingOptions={[]}
+      metadata={metadata}
+      setParameterMapping={jest.fn()}
+      isMobile={false}
+      {...options}
+    />,
+  );
+};
+
+describe("DashCardParameterMapper", () => {
+  it("should render an unauthorized state for a card with no dataset query", () => {
+    setup();
+    expect(getIcon("key")).toBeInTheDocument();
+  });
```

Comment:
Is there anything else except the icon we can look up for? If the test fails, having e.g. a user-facing string would make it much easier to find a broken piece of code

Reply:
I can update the component to expose an aria label. 👍